tests/lib/rules/utils/ast-utils.js

```diff
@@ -1802,4 +1802,62 @@ describe("ast-utils", () => {
             });
         });
     });
+
+    describe("isTopLevelExpressionStatement", () => {
+        it("should return false for a Program node", () => {
+            const node = { type: "Program", parent: null };
+
+            assert.strictEqual(astUtils.isTopLevelExpressionStatement(node), false);
+        });
+
+        it("should return false if the node is not an ExpressionStatement", () => {
+            linter.defineRule("checker", {
+                create: mustCall(context => ({
+                    ":expression": mustCall(node => {
+                        const sourceCode = context.getSourceCode();
+
+                        assert.strictEqual(astUtils.isTopLevelExpressionStatement(sourceCode, node), false);
```

Comment:
```suggestion
                        assert.strictEqual(astUtils.isTopLevelExpressionStatement(node), false);
```

`astUtils.isTopLevelExpressionStatement` takes only 1 argument.